cmake/modules/3.24/FindPostgreSQL.cmake

```diff
@@ -102,7 +102,7 @@ set(PostgreSQL_ROOT_DIR_MESSAGE "Set the PostgreSQL_ROOT system variable to wher
 
 
 set(PostgreSQL_KNOWN_VERSIONS ${PostgreSQL_ADDITIONAL_VERSIONS}
-    "13" "12" "11" "10" "9.6" "9.5" "9.4" "9.3" "9.2" "9.1" "9.0" "8.4" "8.3" "8.2" "8.1" "8.0")
+    "14" "13" "12" "11" "10" "9.6" "9.5" "9.4" "9.3" "9.2" "9.1" "9.0" "8.4" "8.3" "8.2" "8.1" "8.0")
```

Comment:
If there is no other change to this module, I would propose to leave this find module unchanged, in directory `3.20`, and add missing versions via `set(PostgreSQL_ADDITIONAL_VERSIONS "14" CACHE STRING "...")`, just before calling `gdal_check_package`/`find_package`.

Reply:
good point. PR amended